src/message/MessageBrief.js

```diff
@@ -48,9 +48,9 @@ export default class MessageBrief extends React.PureComponent {
       <View style={styles.message}>
         <View style={styles.messageContentWrapper}>
           <TouchableWithoutFeedback onLongPress={onLongPress}>
-            <View style={styles.childrenWrapper}>
```

Comment:
Tried without this change, scrolls are not working all time.
doesn't worked for 
https://chat.zulip.org/#narrow/stream/development.20help/subject/apt.20repo/near/241568
but worked for 
https://chat.zulip.org/#narrow/stream/development.20help/subject/mypy.20imports/near/222405